marc_to_solr/lib/princeton_marc.rb

```diff
@@ -789,35 +789,6 @@ def process_recap_notes record
   item_notes
 end
 
-def find_hathi_by_oclc(oclc)
```

Comment:
@kevinreiss @maxkadel this code is not running the line that was added `return if ENV["RUN_HATHI_COMPARE"].blank?` is actually a switch. Keeping it here doesn't change something.

Reply:
From my POV, if we're not using code, the rest of the code base is likely to drift, and it's going to be hard to maintain it, and hard to know if / when we break it. 

To me, it makes more sense to take it out and know it's in the history, in a context where it was working, rather than to leave it in but non-functional.

Reply:
@maxkadel my response was in regards to the following question https://github.com/pulibrary/bibdata/pull/1907#discussion_r907648284 , https://pulibrary.slack.com/archives/GNN8N3Y6P/p1656417983864719 and that there is no need to move the code somewhere else in the codebase. From the other side you can remove it and github has the history to reveal it at anytime.